pkg/agent/svid/store/service.go

```diff
@@ -107,7 +107,12 @@ func (s *SVIDStoreService) deleteSVID(ctx context.Context, log logrus.FieldLogge
 		return false
 	}
 
-	log = log.WithField(telemetry.SVIDStore, storeName)
+	log = log.WithFields(logrus.Fields{
+		telemetry.SVIDStore: storeName,
+		telemetry.Entry:     entry.EntryId,
```

Comment:
why are you replacing log here?  it is defined above and contains EntryID and SPIFFEID

Reply:
Thanks for catching this. I was somehow confused about missing the entry ID and SPIFFE ID.